C/README.md

```diff
@@ -13,4 +13,4 @@ C is a general-purpose, high-level language that was originally developed by Den
 |    Stack implementation    |                 [link](./StackImplementation.c)                  |
 |  Sum of digits of number   |                  [link](./sumOfNumberDigits.C)                   |
 |  Binary Search Tree        |                  [link](./BinarySearchTree.C)                    |
-|  Floyd's Cycle Detection   |                  [link](./floyds_cycle_detection_for_linked_lists.c)				|
+|  Hash Table Implementation |                  [link](./HashTableImplementation.c)             | 
```

Comment:
By mistake, you removed another contributor entry! Please add it back and your PR is ready to merge!

Reply:
Okay I am on it